src/prog/CalcGW.cpp

```diff
@@ -534,6 +552,16 @@ CLIOptions::CLIOptions(const BSMPT::parser &argparser)
     ss << "--trans_string not set, using default value: perc\n";
   }
 
+  try
+  {
+    UserDefined_PNLO_scaling = argparser.get_value<double>("pnlo_scaling");
```

Comment:
```suggestion
    UserDefined_PNLO_scaling = argparser.get_value<int>("pnlo_scaling");
```
Because it's an `int`.